src/components/MovieDetails/index.tsx

```diff
@@ -354,19 +354,23 @@ const MovieDetails: React.FC<MovieDetailsProps> = ({ movie }) => {
           </div>
           <h1 className="text-2xl lg:text-4xl">
             {data.title}{' '}
-            <span className="text-2xl">({data.releaseDate.slice(0, 4)})</span>
+            {data.releaseDate && (
+              <span className="text-2xl">({data.releaseDate.slice(0, 4)})</span>
+            )}
           </h1>
           <span className="mt-1 text-xs lg:text-base lg:mt-0">
-            {(data.runtime ?? 0) > 0 && (
+            {data.runtime || data.genres.length ? (
```

Comment:
Do we need a conditional operator here?

Reply:
Yeah, otherwise it shows a 0. 🤔

Reply:
You could convert it to a boolean? > 0 or something.

Reply:
Yeah but I think this is cleaner, since I'd have to add that for the `genres` arary length as well otherwise?